doc/ref_model/chapters/chapter01.md

```diff
@@ -132,32 +74,6 @@ This document specifies:
   - **Certification programs**: This defines the requirements for certification and validation programs for both VNFs and NFVI.
```

Comment:
Suggest changing `NFVI` to `cloud infrastructure`

Reply:
done: https://github.com/cntt-n/CNTT/pull/1267/commits/ca8ae50d51bd5331ec3aaa55569f51d2ffa8bf86